backend/src/email/routes/email-campaign.routes.ts

```diff
@@ -637,4 +644,49 @@ router.get('/stats', EmailStatsMiddleware.getStats)
  */
 router.get('/export', EmailStatsMiddleware.getFailedRecipients)
 
+/**
+ * @swagger
```

Comment:
Hmmm swagger documentation doesn't seem to match. Will the `passwordHash` be in the body or params?